modules/architecture-rhcos-updating-bootloader.adoc

```diff
@@ -32,13 +32,20 @@ You can manually install `bootupd` by using the `bootctl` command-line tool.
 # bootupctl status
 ----
 +
-.Example output
+.Example outputs
```

Comment:
Maybe make this "Example output for x86_64" and then for the new block add "Example output for arm64"?